src/assets/styles/30-components/__manifest.scss

```diff
@@ -72,4 +72,3 @@
 @import './src/components/TextInput/text-input';
 @import './src/components/Timeline/timeline';
 @import './src/components/Video/video';
-@import './src/components/WellcomeCollectionBanner/wellcome-collection-banner';
```

Comment:
I assume WellcomeCollectionBanner is now completely unused, hence you removing it?

Reply:
I didn't find it anywhere in the codebase. We are using SiteAlert for Wellcome Collection banner in CR.